Module import error when running with npx

I'm using node 18 for most of my projects and when I run `npx @getalby/pkgzap-cli` I get the following module import error

```
$ npx @getalby/pkgzap-cli
node:internal/process/esm_loader:97
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/xxxxxx/.npm/_npx/3d08cac78fe9456f/node_modules/@scure/bip39/wordlists/english' imported from /home/xxxxxx/.npm/_npx/3d08cac78fe9456f/node_modules/nostr-tools/lib/esm/nostr.mjs
Did you mean to import @scure/bip39/wordlists/english.js?
    at new NodeError (node:internal/errors:405:5)
    at finalizeResolution (node:internal/modules/esm/resolve:324:11)
    at moduleResolve (node:internal/modules/esm/resolve:943:10)
    at defaultResolve (node:internal/modules/esm/resolve:1129:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v18.17.0
```

## Reproduce
 - `docker run --rm node:18 /bin/bash`
 - `npx @getalby/pkgzap-cli`
 
 I believe it could be related to https://github.com/nbd-wtf/nostr-tools/issues/253

--- a/cli/package.json
+++ b/cli/package.json
@@ -23,6 +23,9 @@
     "crypto": "^1.0.1",
     "websocket-polyfill": "^0.0.3"
   },
+  "overrides": {
+    "@scure/bip39": "^1.2.1"
+  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/getAlby/pkgzap.git"

